### PR TITLE
Large hypo fix and new large bottles for chemistry press

### PR DESCRIPTION
**Pull request by MajManatee**
## About The Pull Request

Increases the chemical press capacity to 100 to allow large hypos to be filled fully, and adds a new large bottle, capable of holding 100 units.

As a QOL request, the minimum size for things is now 1u.

## Why It's Good For The Game
why can we only fill large hypos halfway? 
Large bottles great for stuff like blood or synthflesh, or mixtures of chems.
## Proof Of Testing
<details>
<summary>Screenshots/Videos</summary>

Large bottles (Filled and coming from the press)
image

Large hypos working the same
image

image

</details>

## Changelog
:cl:
add: Chemical press can now fill large bottles (100u)
add: Adds large bottles
qol: Chemical press can now fill large hypos fully (100u)
qol: Chemical press can now dispense pills and patches at 1u
/:cl:

### CODE REVIEW

**Comment by Erol509** [comment]
I dont think, a bigger bottle is needed, and its tiny bit of a balance issue. Mainly because you are making bottles out of nothing, just pure power.

**Comment by sippykot** [comment]
i just wish we had bottles in the chemmaster again

**Comment by MajManatee** [comment]
(drafted till I can get the bottles to appear in the UI, they are being mean and refusing to)
Unlikely to affect any balance since this is (currently) the only way to GET a large bottle. Its something that I have wanted after doing a LOT of chem, especially for blood

**Comment by sippykot** [comment]
heyyy, could you also lower the minimum reagent amount to 1? i have no clue why you have to have 5u pills/patches minimum

File: code/modules/plumbing/plumbers/pill_press.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ///the minimum size of a pill or patch
-#define MIN_VOLUME 5
+#define MIN_VOLUME 1
 ///the maximum size a pill or patch can be
-#define MAX_VOLUME 50
+#define MAX_VOLUME 100
```
Comment (Majkl-J) [inline code/modules/plumbing/plumbers/pill_press.dm:4]: Also, won't this cause issues with overfilling pills and patches?

File: icons/obj/medical/chemical.dmi
Comment (Majkl-J) [inline icons/obj/medical/chemical.dmi:1]: Ditto on modularity

File: icons/obj/medical/reagent_fillings.dmi
Comment (Majkl-J) [inline icons/obj/medical/reagent_fillings.dmi:1]: Ditto, modularize

File: code/_globalvars/lists/reagents.dm [hunk 2]
```diff
@@ -44,6 +44,10 @@ GLOBAL_LIST_INIT(reagent_containers, list(
 		/obj/item/reagent_containers/syringe/smartdart
 	)),
 	// SKYRAT EDIT ADDITION END
+	CAT_MEDBOTTLES = typecacheof(list(
+		/obj/item/reagent_containers/cup/bottle/medi,
+		/obj/item/reagent_containers/cup/bottle/large
+	)),
```
[the remark this answers is not in the document]
Comment (MajManatee) [reply code/_globalvars/lists/reagents.dm:50]: not a skyrat change tho. but ok.

**Review by Majkl-J — APPROVED** [review]
Seems to have worked fine when I tested it and modularity is good now

**Comment by MajManatee** [comment]
i should have mentioned that i tested it when I requested a review, and fixed the one thing I forgot

**Pull request merged**